Gemfile

```diff
@@ -2,9 +2,11 @@ source 'https://rubygems.org'
 
 gemspec
 
-case version = ENV['MONGOID_VERSION'] || '~> 6.0.0'
+case version = ENV['MONGOID_VERSION'] || '~> 7.0.0'
 when 'HEAD'
   gem 'mongoid', github: 'mongodb/mongoid'
+when /7/
```

Comment:
Please use `/\b7\./`   The current regex implies 7 can be anywhere in the string, i.e. 1.7.0.

Reply:
Good point.